components.inky.js

```diff
@@ -0,0 +1,364 @@
+function getElementClasses(element) {
+  var classes = [];
+  if (element.attr('class')) {
+    classes = classes.concat(element.attr('class').split(' '));
+  }
+  return classes.join(' ');
+}
+
+module.exports = {
```

Comment:
I'm really sorry, I should have explained my thinking 😞  

For larger projects like this I imagine this to be more of a manifest file. So something like

```
import tbgbuttonJS from './src/components/tbgbutton';

export let tbgbutton = {
  js: tbgbuttonJS
}
```
Not to say it should take that format, thats just an example of my thinking.

Really sorry to mess you about like that!